src/Components/DisplayEvents.js

```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import Event from "./Event";
+
+function DisplayEvents({ sort, search }) {
+  const [events, setEvents] = useState([]);
+
+  // API CALL - first render
+  useEffect(() => {
+    console.log(sort);
+    axios({
+      url: `https://app.ticketmaster.com/discovery/v2/events?apikey=ssi7z7zAxXhwJbRzJY0OsBa6P0feF23r&locale=*&startDateTime=2022-10-30T15:38:00Z&endDateTime=2023-10-31T15:38:00Z&countryCode=CA`,
+      method: "GET",
+      dataResponse: "json",
+      params: {
+        keyword: search,
+        city: sort,
+      },
+    }).then((res) => {
+      setEvents(res?.data?._embedded?.events);
+      console.log(res?.data?._embedded?.events);
+    });
+  }, [sort, search]);
+
+  return (
+    <section className="mainContainer wrapper">
+      <div className="upcomingEvents">
+        <h2>Upcoming events:</h2>
```

Comment:
since right now your default page doesn't show any data on load, maybe the h2 could be a conditional render so that it shows Upcoming events: only when events are loaded, and something else (i.e. instructions for the user) when there are no events showing